robotbase/src/main/java/com/team2357/frc2022/commands/ClimberReleaseLatchOnStrainCommand.java

```diff
@@ -0,0 +1,28 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.frc2022.subsystems.ClimberSubsystem;
+import com.team2357.lib.commands.CommandLoggerBase;
+
+public class ClimberReleaseLatchOnStrainCommand extends CommandLoggerBase{
+    private ClimberSubsystem m_climbSub;
+    private boolean m_isFinished;
+
+    public ClimberReleaseLatchOnStrainCommand(ClimberSubsystem climbSub) {
+        m_climbSub = climbSub;
+        m_isFinished = false;
+        addRequirements(m_climbSub);
+    }
+
+    @Override
+    public void execute() {
+        if(m_climbSub.isClimberGripped()) {
```

Comment:
Do we need this check?

Reply:
Depends on if we want to let the latch release purely on timing, or when the climber arms are trying to pull up the bar.